### ISSUE TITLE
Не ровная склейка env/skybox в OpenGL

### ISSUE THREAD

**Issue by 0x4E69676874466F78** [FWGS/xash3d-fwgs#707]
Заходим на c2a5a и видим такое (зажевало немного края):
![изображение](https://user-images.githubusercontent.com/4449851/146942603-8a94ee15-1d12-4eac-97b9-db0ef076524b.png)
Смешно что баг прямиком из WON халфы:
![изображение](https://user-images.githubusercontent.com/4449851/146942781-c403821e-0951-4e59-9b94-fe62dbe29fa0.png)
При этом в вулкан рендере (сейчас не корректная экспозиция):
![изображение](https://user-images.githubusercontent.com/4449851/146942377-361d2156-8edc-451a-b70a-451b792edb51.png)

Для сравнения кубемапа:
![изображение](https://user-images.githubusercontent.com/4449851/146943734-d877780d-8317-4450-ad45-8fda9fb8a9ba.png)

**Comment by Vladislav4KZ** [comment]
Я конечно замечал это в игре на разных картах, но на скринах разницы в скайбоксе не вижу

**Comment by 0x4E69676874466F78** [comment]
@Vladislav4KZ 
![изображение](https://user-images.githubusercontent.com/4449851/146947814-7c9545dc-afa7-4311-af67-fbda0b2f4bfe.png)
![изображение](https://user-images.githubusercontent.com/4449851/146947968-b3633778-d438-4a42-87b9-910f8a95d8ae.png)

**Comment by a1batross** [comment]
Больше похоже на типичные беды клампа в OpenGL.

**Comment by 0x4E69676874466F78** [comment]
На других картах кстати где-то кубемапы вовсе неправильные сами по себе, имейте это ввиду.

**Comment by Vladislav4KZ** [comment]
> @Vladislav4KZ 
> ![изображение](https://user-images.githubusercontent.com/4449851/146947814-7c9545dc-afa7-4311-af67-fbda0b2f4bfe.png)
> ![изображение](https://user-images.githubusercontent.com/4449851/146947968-b3633778-d438-4a42-87b9-910f8a95d8ae.png)
> 

Да, я понял. Я только хотел кинуть скрин с карты crossfire. Там тоже видны швы, скайбокс "desert"

**Comment by 0x4E69676874466F78** [comment]
На c2a5a скабокс cliff.

**Comment by a1batross** [comment]
Fixed. https://github.com/FWGS/xash3d-fwgs/commit/f9d4ac2771dbf2405ba94331fc3c5229d71cb291
